sktalk/corpus/parsing/cha.py

```diff
@@ -0,0 +1,69 @@
+import re
+import pylangacq
+from ..conversation import Conversation
+from ..utterance import Utterance
+from .parser import Parser
+
+
+class ChaParser(Parser):
+    def parse(self, file):
+        """Parse conversation file in Chat format
+
+        Args:
+            file (str): path to file in .cha format
+
+        Returns:
+            Conversation: A Conversation object representing the conversation in the file.
+        """
+        chatfile = pylangacq.read_chat(file)
+        chat_utterances = chatfile.utterances(by_files=False)
+
+        metadata = ChaParser._get_metadata(chatfile)
+
+        utterances = [ChaParser._to_utterance(
+            row, file) for row in chat_utterances]
+
+        return Conversation(utterances, metadata)
+
+    @staticmethod
+    def _to_utterance(row, file):
+        utterance = Utterance(
+            participant=row.participant,
+            time=row.time_marks,
+            utterance=str(row.tiers),
+            source=file,
+        )
+        utterance.begin, utterance.end = ChaParser._split_time(utterance.time)
+        utterance.utterance = ChaParser._clean_utterance(utterance.utterance)
+        return utterance
+
+    @staticmethod
+    def _get_metadata(chatfile):
+        return chatfile.headers()[0]
+
+    @staticmethod
+    def _split_time(time):
```

Comment:
Is this logic specific to the Cha format? Otherwise, I think it should migrate to a more generic `utils` package.